nixnet/system/databases.py

```diff
@@ -0,0 +1,149 @@
+﻿from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import typing  # NOQA: F401
+
+import six
+
+from nixnet import _funcs
+
+
+class Databases(collections.Mapping):
+    """Database aliases."""
+
+    def __init__(self, handle):
+        self._handle = handle
+
+    def __repr__(self):
+        return 'System.Databases(handle={0})'.format(self._handle)
+
+    def __get_database_list(self, ip_address):
+        alias_buffer_size, filepath_buffer_size = _funcs.nxdb_get_database_list_sizes(ip_address)
+        aliases, filepaths, _ = _funcs.nxdb_get_database_list(ip_address, alias_buffer_size, filepath_buffer_size)
+        return list(zip(aliases.split(","), filepaths.split(",")))
```

Comment:
I don't think doing a split, zip and list is going to be that expensive. Especially when we don't expect users to use a ton of aliases at once. Jeff expects around 100 as a maximum.

Leaving it as is.